exercises/matrix/matrix_test.exs

```diff
@@ -0,0 +1,66 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("matrix.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule MatrixTest do
+  use ExUnit.Case
+
+  @input "1 2 3\n4 5 6\n7 8 9"
+
+  #@tag :pending
+  test "reading from and writing to string" do
+    matrix = MatrixStruct.from_string(@input)
+
+    assert matrix != nil
```

Comment:
I'm not 100% sure that these tests will be be more clarifying rather than confusing. I see where you're going there, but frankly I think we probably just need the assertion currently on line 24 for this first test.